proc_calibrate_clock.py

```diff
@@ -1,48 +0,0 @@
-from pyspecdata import *
```

Comment:
I believe now we have an automated clock correction implemented into the scripts which makes this obsolete?

Reply:
no, this is still an experiment that we want -- please leave it in